master/getting-started/mesos/tutorials/policy/docker-containerizer.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Network Policy (Docker Containerizer)
+---
+
+Since the Docker Containerizer uses standard docker networking, Calico's standard
```

Comment:
I'd probably elaborate a little here.

The Docker Containerizer uses Docker directly to launch tasks.  When using Calico with Mesos, it uses the standard Calico Docker networking and IPAM plugins.  

Refer to the Docker Tutorials, starting with `Security using Calico Profiles` to understand the how to create different Calico networks and modify policy.